terraform/install.sh

```diff
@@ -174,6 +174,26 @@ applyTerraform() {
   terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"
 }
 
+installMonitoring() {
+  external_ip="";
+  while [ -z $external_ip ]; do
+     external_ip=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'); 
+     [ -z "$external_ip" ] && sleep 10; 
```

Comment:
increase the timeout - current interval is not enough to make a difference.

Reply:
Modified the timeout to 30 seconds. Let me know if you think it needs more.

Reply:
Change it to 5 seconds

Reply:
Changed :)